src/newrelic/limits/limits.go

```diff
@@ -31,6 +31,7 @@ const (
 	MaxCustomEvents              = 10 * 1000
 	MaxErrorEvents               = 100
 	MaxSpanMaxEvents             = 10000
+	DefaultMaxLogEvents          = 2000
```

Comment:
This seems like it has the potential to confuse a future dev down the line and potentially introduce conflicts with other values. Is there a use case for this, that is not covered by the existing default value definition? Since MaxLogMaxEvents is defined right below this, and no other event types have defaults defined here, I would tend to caution against it

Reply:
I've reworked the code logic to not need a default value in commit [c29ecf](https://github.com/newrelic/newrelic-php-agent/pull/497/commits/c29ecf6c529a95ef21c031418ff0bcbd42a3860f).

Reply:
Nice!